README.md

```diff
@@ -1,9 +1,33 @@
-# http-apis.github.io
+# Hydra Ecosystem Documentation
 
-Main documentation entrypoint for Hydra Ecosystem's tools. 
+This is the source-code behind [hydraecosystem.org](https://hydraecosystem.org), the official documentation of the Hydra Ecosystem toolset.
 
-*This website is readable at [hydraecosystem.org](https://hydraecosystem.org)
+## Viewing the Documentation Offline
 
-##### Running the website locally:
+The documentation is hosted on [GitHub Pages](https://pages.github.com) and uses the static site generator named [Jekyll](https://jekyllrb.com). If you want to see what your modifications to the documentation would look like, before you make a PR, please run the following commands:
 
-This is a Github Page, If wanting to create/check how documentation modifications look locally, please follow the [official github tutorial to run it locally](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll).
\ No newline at end of file
+- Check if you have Jekyll installed. if not, follow the [installation guide](https://jekyllrb.com/docs)
+```bash
+jekyll -v
+```
+- Start a live-server which serves statically generated content, changes made will make real-time changes to it.
+```bash
+bundle exec jekyll serve
+```
+
+**Note:** Just running the `jekyll` command without any arguments, might build a static website in html form with-in the `public` directory, please make sure not to include this in your PR.
+
+# Contributing
+
+Hydra Ecosystem is an Open Soure project and we have our source code licensed under the permissible MIT license, please make any contribution to the project by opening an issue and a subsequent PR to solve the same. 
+
+We are part of the first ever **Google *sponsored* Season of Docs** program. If you are a technical writer or someone with decent writing skills and want to start contributing, and you know how to write crisp documentation, please consider taking part in Google Season of Docs 2019 with us!
```

Comment:
I might be wrong but I don't think we should add stuff like this in the README. The README should be strictly about the repository, if we add information about the project and GSoD, we will have to keep that up to date alone with the pages on the site. Wouldn't that then be redundant?

Reply:
You are right, let me fix that :+1: